stubs/iterable.stub

```diff
@@ -157,15 +157,19 @@ class ArrayIterator implements SeekableIterator, ArrayAccess, Countable
  */
 class RecursiveIteratorIterator
 {
+	const LEAVES_ONLY=0;
+	const SELF_FIRST=1;
+	const CHILD_FIRST=2;
+	const CATCH_GET_CHILD=16;
 
 	/**
 	 * @param T $iterator
-	 * @param int $mode
-	 * @param int $flags
+	 * @param self::LEAVES_ONLY|self::SELF_FIRST|self::CHILD_FIRST $mode
```

Comment:
I'd put this in `resources/functionMap_bleedingEdge.php`.

Reply:
initially I tried that, too. turns out I need a mix of both, as the constants were missing on the stub.

Reply:
I don't understand.

Reply:
in the first iteration of this PR I tried adding it to `resources/functionMap_bleedingEdge.php` and did not succeed. now with the PR as is it works

Reply:
I'm interested in why you didn't succeeed. It should be fine.